app/models/mixins/vim_connect_mixin.rb

```diff
@@ -7,9 +7,9 @@ def connect(options = {})
 
     options[:fault_tolerant] = true unless options.key?(:fault_tolerant)
```

Comment:
Wondering if we can get rid of `:fault_tolerant` and just assume yes, no one is sending `connect(:fault_tolerant => false)`...

Reply:
done